_docs/operations/users.md

```diff
@@ -72,3 +72,18 @@ To delete a user, run:
 ```bash
 $ storageos user rm jim
 ```
+## Altering the StorageOS API account
+The StorageOS API account is defined by the [storageos-api
+secret](/docs/platforms/kubernetes/install/1.13#create-a-secret) that is
```

Comment:
This is defined as an env variable. Might be interesting mentioning that it can be specified when deploying StorageOS

Reply:
I'm not sure where I should add that in. I don't think it's a good idea to add it to the _includes/operator/install.md file as it muddies the water over what to do. It's documented in the Cluster Operator Config so I think that's enough. 

@actionbuddha thoughts?

Reply:
Agreed, think simplicity is good at this point.